kpops/components/base_components/pipeline_component.py

```diff
@@ -52,25 +50,21 @@ class PipelineComponent(BaseDefaultsComponent):
         description=describe_object(__doc__),
         exclude=True,
     )
-    name: str = Field(default=..., description="Component name")
+    name: str = Field(default=..., description=describe_attr("name", __doc__))
+    prefix: str = Field(
+        default="${pipeline_name}-",
+        description=describe_attr("prefix", __doc__),
+    )
     from_: FromSection | None = Field(
         default=None,
         alias="from",
         title="From",
-        description=describe_attr("from_", __doc__),
-    )
-    app: object | None = Field(
```

Comment:
Will this break things?

Reply:
I'll test it. It seems like this attribute made it here by mistake or it stems from an old workaround

Reply:
tested ✔️